org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractLinkedEditingIntegrationTest.java

```diff
@@ -32,6 +32,10 @@ public void process(XtextResource state) throws Exception {
 		});
 	}
 	
+	/**
+	 * deprecated, use the inherited {@link AbstractWorkbenchTest#waitForEventProcessing} method instead.
```

Comment:
Shouldn't "deprecated" in the javadoc be an annotation as well?

Reply:
Yes, indeed. It should be an annotation. I have just updated the PR.